### ISSUE TITLE
Unable to use keyboard navigation to switch between scenes

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#573]
**Test device**
MacBook Air M1 chip

**Operating System**
14.3.1

**Browser**
Safari 17.3.1 and Chrome

**Problem description**
Seen while testing https://github.com/phetsims/qa/issues/1039 but unrelated to that MR: 
In published and on main, I am unable to move between the One Balloon and Two Balloon scenes using keyboard navigation (arrow keys). This occurs with both safari and chrome. 

I'm not seeing this issue when moving between the other radio button group (show charges) or when moving between radio buttons in Acid Base Solutions.

**Comment by Nancy-Salpepi** [comment]
Actually @KatieWoe pointed out that you can switch between them by pressing the Space Bar, but that isn't doesn't match the keyboard dialog:
<img width="466" alt="Screenshot 2024-02-14 at 1 55 03 PM" src="https://github.com/phetsims/balloons-and-static-electricity/assets/87318828/7aa3df90-2bab-4f36-b8f9-637f78f1d5fe">

**Comment by terracoda** [comment]
@Nancy-Salpepi, under the hood, that interaction is a checkbox (or switch), named "Two-balloon Experiment". It's not a radio button group with two radio buttons.

We explored several different visual presentations which nobody liked, and we explored several description designs for a radio button group, none of which described the entrance of the green balloon to my satisfaction. 

So, the design team made a compromise to keep the two-button visual, but give it a single bold outline with keyboard focus and mouse hover. If it were implemented as a true radio button group it would have a thinner group highlight and the bolder focus highlight would go around a single button.

With mouse, it doesn't matter where you click, a click toggles the state of the checkbox.

It's working as designed.

**Comment by Nancy-Salpepi** [comment]
Thanks for the explanation Taliesin!
